### PR TITLE
fix bad array generated by tribe_get_organizer_ids sometimes

### PR DESCRIPTION
**Pull request by zbtirrell**
See: https://central.tri.be/issues/39868

### CODE REVIEW

File: src/functions/template-tags/organizer.php [hunk 1]
```diff
@@ -55,6 +55,15 @@ function tribe_get_organizer_ids( $event_id = null ) {
 				$organizer_ids[] = $event_id;
 			} else {
 				$organizer_ids = tribe_get_event_meta( $event_id, '_EventOrganizerID', false );
+
+				// for some reason we store a blank "0" element in this array.
+				// let's scrub this garbage out
+				if ( is_array( $organizer_ids ) ) {
+					$key = array_search( '0', $organizer_ids );
```
Comment (bordoni) [inline src/functions/template-tags/organizer.php]: Can't we just `array_filter`?

Comment (zbtirrell) [inline src/functions/template-tags/organizer.php]: `array_filter` requires a callback which feels less awesome to me, but I'm open to it if in general we think it is better

Comment (bordoni) [inline src/functions/template-tags/organizer.php]: if you pass just `array_filter( (array) $organizer_ids );` and 0 would be excluded, I'm like 90% sure of it...

Comment (zbtirrell) [inline src/functions/template-tags/organizer.php]: I tested with this:
```
<?php
$bad_array = array('0','monkey');
$bad_array = array_filter( (array) $bad_array );
print_r($bad_array);
```
sure enough, it works as you suggest.  Changing.

**Comment by bordoni** [comment]
:wink:

**Pull request merged**